### ISSUE TITLE
Learning stucks?

### ISSUE THREAD

**Issue by 8ANDfinalrule** [halwai/bird_classification#10]
Hey, did you happen to have a problem that a network stucks around loss=5.3? When I try to finetune only fully connected layers  the network doesn't seem to learn.

**Comment by halwai** [comment]
Could you please tell the exact parameters being invoked?

**Comment by 8ANDfinalrule** [comment]
Hi, I increased learning rate a little bit and got the effects. Maybe I didn't wait long enough to see network learning. 
What accuracy on test set did you got on Caltech birds dataset?

**Comment by halwai** [comment]
I got around 52%.
